---
abstract: High-throughput sequencing technologies have greatly facilitated microbiome research and have generated a large volume of microbiome data with the potential to answer key questions regarding microbiome assembly, structure and function. Cluster analysis aims to group features that behave similarly across treatments, and such grouping helps to highlight the functional relationships among features and may provide biological insights into microbiome networks. However, clustering microbiome data are challenging due to the sparsity and high dimensionality. We propose a model-based clustering method based on Poisson hurdle models for sparse microbiome count data. We describe an expectation-maximization algorithm and a modified version using simulated annealing to conduct the cluster analysis. Moreover, we provide algorithms for initialization and choosing the number of clusters. Simulation results demonstrate that our proposed methods provide better clustering results than alternative methods under a variety of settings. We also apply the proposed method to a sorghum rhizosphere microbiome dataset that results in interesting biological findings.
abstract_short: ""
authors:
- Zhili Qiao
- Elle Barnes
- Susannah Tringe
- Daniel P Schachtman
- Peng Liu 
date: 2023-01-01
header:
  caption: 'Feature selection and causal analysis for microbiome studies in the presence of confounding using standardization'
  image: headers/bubbles-wide.jpg
image_preview: ""
math: true
publication: In *Bioinformatics*.
publication_short: In *Bioinformatics*
publication_types:
- "2"
selected: false
title: "Z Qiao*, E Barnes, S Tringe, D Schachtman, P Liu* (2023). Poisson hurdle model-based method for clustering microbiome features. Bioinformatics."
url_custom:
- name: Link
  url: https://doi.org/10.1093/bioinformatics/btac782
url_code: ""
url_dataset: ""
url_pdf: ""
url_project: ""
url_slides: ""
url_video: ""
---
