---
abstract: RNA-sequencing (RNA-seq) technologies have revolutionized the way that agricultural biologists study gene expression as well as generated a tremendous amount of data waiting for analysis. Detecting differentially expressed genes is one of the fundamental steps in RNA-seq data analysis. In this paper, we model the count data from RNA-seq experiments with a Poissonâ€“Gamma hierarchical model, or equivalently, a negative binomial model. We derive a semi-parametric Bayesian approach with a Dirichlet process as the prior model for the distribution of fold changes between the two treatment means. An inference strategy using Gibbs algorithm is developed for differential expression analysis. The results of several simulation studies show that our proposed method outperforms other methods including the popularly applied edgeR and DESeq methods. We also discuss an application of our method to a dataset that compares gene expression between bundle sheath and mesophyll cells in maize leaves. Supplementary materials accompanying this paper appear online.
abstract_short: ""
authors:
- Fangfang Liu
- Chong Wang
- Peng Liu
date: 2015-12-20
header:
  caption: 'A Semi-parametric Bayesian Approach for Differential Expression Analysis of RNA-seq Data'
  image: headers/bubbles-wide.jpg
image_preview: ""
math: true
publication: In *Journal of Agricultural, Biological, and Environmental Statistics*.
publication_short: In *JABES*
publication_types:
- "2"
selected: true
title: "Liu F., Wang C.,  and Liu P.* (2015). A Semi-parametric Bayesian Approach for Differential Expression Analysis of RNA-seq Data. Journal of Agricultural, Biological, and Environmental Statistics. 20(4): 555-576."
url_code: ''
url_custom:
- name: Link
  url: https://link.springer.com/article/10.1007/s13253-015-0227-0
url_dataset: ''
url_pdf: 'https://link.springer.com/article/10.1007/s13253-015-0227-0'
url_project: ''
url_slides: ''
url_video: ''
---

