---
abstract: "A central goal of RNA sequencing (RNA-seq) experiments is to detect differentially expressed genes. In the ubiquitous negative binomial model for RNA-seq data, each gene is given a dispersion parameter, and correctly estimating these dispersion parameters is vital to detecting differential expression. Since the dispersions control the variances of the gene counts, underestimation may lead to false discovery, while overestimation may lower the rate of true detection. After briefly reviewing several popular dispersion estimation methods, this article describes a simulation study that compares them in terms of point estimation and the effect on the performance of tests for differential expression. The methods that maximize the test performance are the ones that use a moderate degree of dispersion shrinkage: the DSS, Tagwise wqCML, and Tagwise APL. In practical RNA-seq data analysis, we recommend using one of these moderate-shrinkage methods with the QLShrink test in QuasiSeq R package."
abstract_short: ""
authors:
- William M. Landau 
- Peng Liu
date: 2013-12-09
header:
  caption: "Dispersion Estimation and Its Effect on Test Performance in RNA-seq Data Analysis: A Simulation-Based Comparison of Methods"
  image: headers/bubbles-wide.jpg
image_preview: ""
math: true
publication: In *PLoS ONE*.
publication_short: In *PLoS ONE*
publication_types:
- "2"
selected: false
title: "W. Landau, and P. Liu (2013). Dispersion Estimation and Its Effect on Test Performance in RNA-seq Data Analysis: A Simulation-Based Comparison of Methods. PLoS ONE, 8(12):e81415."
url_custom:
- name: Link
  url: http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0081415
url_code: ""
url_dataset: ""
url_pdf: ""
url_project: ""
url_slides: ""
url_video: ""
---
